pkg/config/analysis/msg/messages.yaml

```diff
@@ -644,3 +644,13 @@ messages:
         type: "[]string"
       - name: namespace
         type: string
+
+  - name: "DeprecatedLightstepProvider"
```

Comment:
We already have a general "Deprecated" message, do we need another?

Reply:
I will revise later.